src/user-statuses/UserStatusScreen.js

```diff
@@ -1,29 +1,51 @@
 /* @flow strict-local */
+// $FlowFixMe[untyped-import]
+import isEqual from 'lodash.isequal';
 import React, { useState, useContext, useCallback } from 'react';
 import type { Node } from 'react';
-import { FlatList, View } from 'react-native';
-import { TranslationContext } from '../boot/TranslationProvider';
-import { createStyleSheet } from '../styles';
+import { FlatList, View, Pressable } from 'react-native';
 
+import { TranslationContext } from '../boot/TranslationProvider';
+import { createStyleSheet, BRAND_COLOR, HIGHLIGHT_COLOR } from '../styles';
 import type { RouteProp } from '../react-navigation';
 import type { AppNavigationProp } from '../nav/AppNavigator';
 import { useSelector } from '../react-redux';
 import Input from '../common/Input';
+import EmojiInput from './EmojiInput';
+import type { Value as EmojiInputValue } from './EmojiInput';
+import {
+  emojiTypeFromReactionType,
+  reactionTypeFromEmojiType,
+  parseUnicodeEmojiCode,
+} from '../emoji/data';
 import SelectableOptionRow from '../common/SelectableOptionRow';
 import Screen from '../common/Screen';
 import ZulipButton from '../common/ZulipButton';
-import { getAuth, getOwnUserId } from '../selectors';
+import { getZulipFeatureLevel, getAuth, getOwnUserId } from '../selectors';
 import { getUserStatus } from './userStatusesModel';
-import { IconCancel, IconDone } from '../common/Icons';
+import type { UserStatus } from '../api/modelTypes';
+import { Icon } from '../common/Icons';
 import statusSuggestions from './userStatusTextSuggestions';
 import * as api from '../api';
 
 const styles = createStyleSheet({
-  statusTextInput: {
+  inputRow: {
+    flexDirection: 'row',
     margin: 16,
   },
-  buttonsWrapper: {
-    flexDirection: 'row',
+  statusTextInput: {
+    flex: 1,
+  },
+  clearButton: {
+    // Min touch-target size
+    minWidth: 48,
+    minHeight: 48,
+
+    alignItems: 'center',
+    justifyContent: 'center',
+
+    // To match margin between the emoji and text inputs
+    marginLeft: 4,
   },
   button: {
     flex: 1,
```

Comment:
Is this `flex: 1` still doing something? I'd guess that it should be deleted when the enclosing "button wrapper" View is deleted -- the one that had `flexDirection: 'row'` because there were two buttons sharing the horizontal space.